kuma/search/commands.py

```diff
@@ -3,47 +3,58 @@
 
 from django.conf import settings
 from django.db import reset_queries
+
 from elasticsearch.exceptions import NotFoundError
 
+from kuma.wiki.search import WikiDocumentType
+from kuma.wiki.tasks import index_documents
+
 from .decorators import requires_good_connection
-from .index import (get_indexing_es, get_index, recreate_index, get_indexable,
-                    index_chunk, get_indexes, delete_index_if_exists,
-                    get_index_stats)
+from .index import recreate_index
+from .models import Index
 from .utils import chunked, format_time
 
+
 log = logging.getLogger('kuma.search.commands')
 
 
 @requires_good_connection
-def es_reindex_cmd(percent=100, mapping_types=None,
-                   chunk_size=1000, index=None):
+def es_reindex_cmd(chunk_size=1000, index=None, percent=100):
```

Comment:
Let's rename that to `reindex_all` and move into the `WikiDocumentType` class (or a parent class if you want to keep it agnostic)